public/locales/cs/translation.json

```diff
@@ -68,5 +68,17 @@
       "title": "Manuálně definovaná intenzita poruch",
       "description": "Intenzita poruch nastavená manuálně týmem AVA na základě zkušeného inženýrského úsudku."
     }
+  },
+  "faultTreeOverviewTable": {
+    "name": "Jméno",
+    "aircraftType": "Typ letadla",
+    "ata": "ATA",
+    "calculatedFailureRate": "Vypočtená int. poruch",
+    "fhaBasedFailureRate": "Založená int. poruch",
```

Comment:
Int. poruch založená na FHA
@blcham

Reply:
@grotskat this PR is already merged and we need to review all translations. Please resolve it in the issue https://github.com/kbss-cvut/fta-fmea-ui/issues/261